tests/test_download.py

```diff
@@ -51,49 +52,7 @@ def reload_downloaders(urls, try_downloader=None):
         views.try_downloader = SymbolDownloader([try_downloader])
 
 
-def test_client_happy_path(client, gcsmock, metricsmock):
-    reload_downloaders("https://storage.googleapis.example.com/private/prefix/")
-
-    mock_bucket = gcsmock.MockBucket()
-
-    def mock_get_bucket(bucket_name):
-        assert bucket_name == "private"
-        return mock_bucket
-
-    gcsmock.get_bucket = mock_get_bucket
-
-    def mock_get_blob(key):
-        if key == "prefix/v0/xul.pdb/44E4EC8C2F41492B9369D6B9A059577C2/xul.sym":
-            # As if it exists happily
-            blob = gcsmock.mock_blob_factory(key)
-            blob.public_url = "https://googleapis.example.com/private/" + key
-            return blob
-
-        raise NotImplementedError(key)
-
-    mock_bucket.get_blob = mock_get_blob
-
-    url = reverse(
-        "download:download_symbol",
-        args=("xul.pdb", "44E4EC8C2F41492B9369D6B9A059577C2", "xul.sym"),
-    )
-    response = client.get(url)
-    assert response.status_code == 302
-    parsed = urlparse(response["location"])
-    assert parsed.netloc == "googleapis.example.com"
-    # the pre-signed URL will have the bucket in the path
-    assert parsed.path == (
-        "/private/prefix/v0/xul.pdb/44E4EC8C2F41492B9369D6B9A059577C2/xul.sym"
-    )
-    response = client.head(url)
-    assert response.status_code == 200
-    assert response.content == b""
-
-    assert response["Access-Control-Allow-Origin"] == "*"
-    assert response["Access-Control-Allow-Methods"] == "GET"
-
-
-def test_client_happy_path_s3(client, botomock, metricsmock, settings):
+def test_client_happy_path(client, botomock):
```

Comment:
In many cases, a test using GCS is followed by a test using S3. It is enough to delete the GCS test and rename the S3 test to drop ``_s3`` from the function name.

Reply:
:+1: